Gruntfile.js

```diff
@@ -22,6 +22,23 @@ module.exports = function (grunt) {
 
     var mode = grunt.option("mode") || "dev";
 
+    var copyFiles = [
+        "*.{ico,png,txt}",
+        "bower_components/**/*.{ttf,woff,woff2,svg,eot}",
+        "bower_components/bootstrap/dist/css/bootstrap.min.css",
+        "bower_components/angular-i18n/**.js",
+        "components/**/*.html",
+        "components/**/assets/**/*",
+        ".htaccess",
```

Comment:
@lizardK @frisouilles @blaryjp Is there any reason to copy `.htaccess` file? This one does not exist in our this repository.